pylatex/utils.py

```diff
@@ -131,7 +136,10 @@ def fix_filename(path):
     file_parts = filename.split('.')
 
     if len(file_parts) > 2:
-        filename = '{' + '.'.join(file_parts[0:-1]) + '}.' + file_parts[-1]
+        if os.name == 'posix':
+            filename = '{' + '.'.join(file_parts[0:-1]) + '}.' + file_parts[-1]
+        else:
+            filename = '.'.join(file_parts[0:-1]) + '.' + file_parts[-1]
```

Comment:
This seems like a very convoluted way to make the filename end up in the same way that it already was. I think it makes more sense to change this:
```python
len(file_parts) > 2
```
To:
```python
os.name == 'posix' and len(file_parts) > 2
```

And keep the rest at it originally was (no extra `else` block)